From 13a9cf10eb1836be46f609004950968a1dc20863 Mon Sep 17 00:00:00 2001
From: Peter Lemenkov <lemenkov@gmail.com>
Date: Sun, 13 Feb 2011 14:52:57 +0300
Subject: [PATCH 6/8] Fixes for system-wide ibrowse

Signed-off-by: Peter Lemenkov <lemenkov@gmail.com>

Conflicts:
	src/couchdb/couch_rep.erl
	src/couchdb/couch_rep_changes_feed.erl
	src/couchdb/couch_rep_httpc.erl
---
 src/couchdb/couch_api_wrap_httpc.erl   | 2 +-
 src/couchdb/couch_httpd_proxy.erl      | 2 +-
 src/couchdb/couch_replicator_utils.erl | 2 +-
 3 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/src/couchdb/couch_api_wrap_httpc.erl b/src/couchdb/couch_api_wrap_httpc.erl
index d05eec7..0030615 100644
--- a/src/couchdb/couch_api_wrap_httpc.erl
+++ b/src/couchdb/couch_api_wrap_httpc.erl
@@ -14,7 +14,7 @@
 
 -include("couch_db.hrl").
 -include("couch_api_wrap.hrl").
--include("../ibrowse/ibrowse.hrl").
+-include_lib("ibrowse/include/ibrowse.hrl").
 
 -export([setup/1]).
 -export([send_req/3]).
diff --git a/src/couchdb/couch_httpd_proxy.erl b/src/couchdb/couch_httpd_proxy.erl
index dec3f55..eb037b4 100644
--- a/src/couchdb/couch_httpd_proxy.erl
+++ b/src/couchdb/couch_httpd_proxy.erl
@@ -14,7 +14,7 @@
 -export([handle_proxy_req/2]).
 
 -include("couch_db.hrl").
--include("../ibrowse/ibrowse.hrl").
+-include_lib("ibrowse/include/ibrowse.hrl").
 
 -define(TIMEOUT, infinity).
 -define(PKT_SIZE, 4096).
diff --git a/src/couchdb/couch_replicator_utils.erl b/src/couchdb/couch_replicator_utils.erl
index bd590ed..934e5ce 100644
--- a/src/couchdb/couch_replicator_utils.erl
+++ b/src/couchdb/couch_replicator_utils.erl
@@ -21,7 +21,7 @@
 -include("couch_db.hrl").
 -include("couch_api_wrap.hrl").
 -include("couch_replicator.hrl").
--include("../ibrowse/ibrowse.hrl").
+-include_lib("ibrowse/include/ibrowse.hrl").
 
 -import(couch_util, [
     get_value/2,
-- 
1.8.1

